lib/auth/password_test.go

```diff
@@ -237,37 +239,54 @@ func TestServer_ChangePassword(t *testing.T) {
 	password := mfa.Password
 
 	tests := []struct {
-		name    string
-		newPass string
-		device  *TestDevice
+		name                        string
+		oldPass                     string
+		newPass                     string
+		userVerificationRequirement string
+		device                      *TestDevice
 	}{
 		{
-			name:    "OK TOTP-based change",
-			newPass: "llamasarecool11",
-			device:  mfa.TOTPDev,
+			name:                        "OK TOTP-based change",
+			oldPass:                     password,
+			newPass:                     "llamasarecool11",
+			userVerificationRequirement: "discouraged",
+			device:                      mfa.TOTPDev,
 		},
 		{
-			name:    "OK Webauthn-based change",
-			newPass: "llamasarecool13",
-			device:  mfa.WebDev,
+			name:                        "OK Webauthn-based change",
+			oldPass:                     "llamasarecool11",
+			newPass:                     "llamasarecool13",
+			userVerificationRequirement: "discouraged",
```

Comment:
```suggestion
```

Same. If you want to explicitly check "discouraged" I suggest adding a new test scenario, distinct from the existing ones.